src/admin/class-sesamy-settings-admin.php

```diff
@@ -65,6 +65,7 @@ public function add_settings() {
 				'name'      => 'sesamy_lock_mode',
 				'options'   => array(
 					'embed'     => 'Embed',
+					'encode'     => 'Encode',
```

Comment:
if we want to be really picky about the weird wordpress formatting we would add a space on all the other lines in the array so that the `=>`'s line up 🙃 

Im just thinking since this is kind of open source 🙂 

At some point in the past i had prettier running in vscode with all these weird wordpress formatting rules